highCard.rb

```diff
@@ -0,0 +1,29 @@
+
+puts "Player 1, what is your name? #{input = @playerOneName}"
+puts "Player 2, what is your name? #{input = @playerTwoName}"
+
+suit = [clubs, diamonds, hearts, spades]
+
+rank = rank.new
+%w(ace king queen jack).each_with_index {|item, index|
+  rank[item] = index
+}
+
+@jack = 11
+@queen = 12
+@king = 13
+@ace = 14
+@rank = [2, 3, 4, 5, 6, 7, 8, 9, 10, @jack, @queen, @king, @ace]
+
+def play ()
+  @cardOne = rank[rand]
+  @cardTwo = rank[rand]
+  puts @playerOneName + ", you were dealt the #{@cardOne} of + #{@suit[rand]}"
+  puts @playerTwoName + ", you were dealt the #{@cardTwo} of + #{@suit[rand]}"
+  if cardOne > cardTwo
```

Comment:
`@` behaves just like any other character. For instance, lets say you instead used the letter `z`. So you'd have something like this:

```rb
zcardOne = rank[rand]
zcardTwo = rank[rand]
# ...
if cardOne > cardTwo
```

You wouldn't expect that `if` to work, since it's calling some completely separate variables.